DependencyInjection/RMSPushNotificationsExtension.php

```diff
@@ -97,11 +97,18 @@ protected function setAndroidConfig(array $config)
 
         // GCM
         $this->container->setParameter("rms_push_notifications.android.gcm.enabled", isset($config["android"]["gcm"]));
-        if (isset($config["android"]["gcm"])) {
-            $this->container->setParameter("rms_push_notifications.android.gcm.api_key", $config["android"]["gcm"]["api_key"]);
-            $this->container->setParameter("rms_push_notifications.android.gcm.use_multi_curl", $config["android"]["gcm"]["use_multi_curl"]);
-            $this->container->setParameter('rms_push_notifications.android.gcm.dry_run', $config["android"]["gcm"]["dry_run"]);
-        }
+//        if (isset($config["android"]["gcm"])) {
```

Comment:
I'm not sure on the reasoning behind this.

If the user is not interested in GCM configuration (for example, they are only using iOS push notifications), this forces them to supply configuration options. That is why the conditional check is in place, to enable them to leave out the android.gcm key and similar (see the `Configuration` class which permits the key to be unset/not supplied if not needed).

Commenting this check out will force the user to supply dummy details. Therefore I'd like this to be left in.

This is the same for the FCM change on line 108.